runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsPlugin.java

```diff
@@ -72,6 +75,7 @@
 	private boolean barsDisplayed;
 
 	private int lastCombatActionTickCount;
+	boolean inLMS;
```

Comment:
```suggestion
	@Getter(AccessLevel.PACKAGE)
	private boolean inLMS;
```

and change reads to `plugin.isInLms()`